thirdeye/thirdeye-pinot/src/main/resources/scripts/getGraphPnj.js

```diff
@@ -63,29 +42,58 @@ page.open(options.anomalyRoute, function(status) {
   }
 });
 
+page.onCallback = function(data) {
+  console.log(data);
```

Comment:
Do you want these console logs to remain?

Reply:
the script also has a bunch of `system.stderr.writeLine` which acts like console.log. This is pretty useful for debugging purposes as phantomJS is a headless browser. 
Let me know if you disagree!